src/layers/xr-layer/xr-layer.js

```diff
@@ -1,11 +1,13 @@
+/* eslint-disable prefer-destructuring */
```

Comment:
There are a lot of ignores here... If it's a pain to fix now, we could just ignore this file, and make sure it is on our TODO list to finish it up... or if you think `prefer-destructuring` is overrated, make a case for turning it off.

Reply:
It's not that it's a pain - it's that this is the way it works. I suppose we could `return true` from the `loadTextures` function?  I don't mind the ignores.

Reply:
Add comment about ignores